dump/dump-to-bucket.py

```diff
@@ -109,16 +109,16 @@ def scrape(book, host, visited_locs=VISITED_LOCS_MARKER):
 
     try:
         id, version = split_ident_hash(book)
+        info(f'Requested Version: {T.bold}{id}@{version}{T.normal}')
     except cnxcommon.ident_hash.IdentHashMissingVersion:
+        info('No version was Given. Retrieving Latest Version')
         id = book
-    base_url = f'https://{host}/contents'
-
-    # Request the latest version
-    url = f'{base_url}/{id}.json'
-    resp = session.get(url)
-    version = resp.json()['version']
-
-    info(f'latest version of requested book: {T.bold}{id}@{version}{T.normal}')
+        base_url = f'https://{host}/contents'
+        url = f'{base_url}/{id}.json'
+        resp = session.get(url)
+        latest_version = resp.json()['version']
```

Comment:
@c0ldcalamari - Is the use of latest_version here perhaps left over from some debugging? It seems like you can just assign directly to `version = resp.json()['version']` ?

Reply:
My intentions here were to be explicit as to what version we were getting, but I can see how it is also redundant, since I have `info('No version was Given. Retrieving Latest Version')` a couple lines above. Will remove, thank you!